packages/terra-filter-view/src/_Frame.module.scss

```diff
@@ -0,0 +1,181 @@
+@import '~terra-mixins';
+
+:local {
+  .select-parent {
+    display: flex;
+    flex-direction: column;
+    height: 100%;
+  }
+
+  .select {
+    background-color: var(--terra-filter-background-color, #fff);
+    color: var(--terra-filter-color, #1c1f21);
+    display: inline-flex;
+    font-size: var(--terra-filter-font-size, 1.143rem);
+    line-height: var(--terra-filter-line-height, 1.5);
+    position: relative;
+  }
+
+  .display {
+    overflow: hidden; // IE 10
+  }
+
+  .button {
+    border: 0.071em solid #dedfe0;
+    border-bottom-left-radius: 0;
+    border-bottom-right-radius: 0.25em;
+    border-top-left-radius: 0;
+    border-top-right-radius: 0.25em;
+    display: inline-block;
+    flex: 0 0 auto;
+    margin: 0;
+
+    &:hover {
+      background-color: #b9bbbc;
+      border-color: #c8cacb;
+    }
+
+    &:active {
+      background-color: #6f7477;
+      border-color: #52585b;
+      color: #fff;
+    }
+  }
+
+  .clear-button {
+    color: var(terra-filter-clear-button-color, #b9bbbc);
+    margin-left: 0.3571428571rem;
+    margin-right: 0.3571428571rem;
+    position: absolute;
+    right: 30px;
+    top: 7px;
+  }
+
+  .search-input,
+  .search-input:focus {
+    background: transparent;
+    border: 0;
+    color: inherit;
+    flex: 1 1 auto;
+    font-size: var(--terra-filter-search-font-size, 1.143rem);
+    height: 1.375em;
+    margin-top: var(--terra-filter-search-margin-top, 0.21428571428571427rem);
+    max-width: 100%; // IE 10
+    min-width: 0; // IE 11
+    opacity: 1;
+    outline: none;
+    text-overflow: ellipsis;
+    width: auto;
+
+  }
+
+  .right-padding {
+    // Add padding to avoid text from hiding under clear-search button
+    padding: 0 26px 0 0;
+  }
+
+  // Hides the clear buttons in IE.
+  .search-input::-ms-clear {
+    display: none;
+    height: 0;
+    width: 0;
+  }
+
+  .search-input::placeholder,
+  .placeholder {
+    color: var(--terra-filter-search-placeholder-color, #85898b);
+    font-size: var(--terra-filter-search-placeholder-font-size, 1.143rem);
+    overflow: hidden;
+    text-overflow: ellipsis;
+    white-space: nowrap;
+  }
+
+  .box {
+    border: var(terra-filter-list-border, 1px solid #dedfe0);
+    flex: 1;
+    margin-top: var(terra-filter-list-margin-top, 0.5rem);
+    overflow: auto;
+  }
+
+  .is-disabled {
+    cursor: default;
+    opacity: var(--terra-filter-disabled-opacity, 0.25);   /*this is overall – the full component*/
+    pointer-events: none;
+
+    ::placeholder {
+      color: var(--terra-filter-disabled-placeholder-color, #868a8c);
+    }
+
+    .button {
+      background-color: var(--terra-filter-disabled-button-background-color, #e2e3e4);
+      border: var(--terra-filter-disabled-border, 1px solid #85898b);
+      opacity: var(--terra-filter-disabled-button-opacity, 0.5);
+    }
+
+    .box {
+      background-color: var(--terra-filter-disabled-background-color, rgba(111, 116, 119, 0.2));
```

Comment:
I'd rename this to `--terra-filter-disabled-list-background-color` to match the variable names on line 94 and 96